lib/node_modules/@stdlib/number/float16/base/to-word/lib/main.js

```diff
@@ -0,0 +1,194 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var NINF = require( '@stdlib/constants/float16/ninf' );
+var PINF = require( '@stdlib/constants/float16/pinf' );
+var isnanf = require( '@stdlib/math/base/assert/is-nanf' );
+var absf = require( '@stdlib/math/base/special/absf' );
+var exponent = require( '@stdlib/number/float32/base/exponent' );
+var isNegativeZerof = require( '@stdlib/math/base/assert/is-negative-zerof' );
+var toWordf = require( '@stdlib/number/float32/base/to-word' );
+
+
+// VARIABLES //
+
+var UINT16_NAN = 0x7E00;                    // 0x7E00 = 32256 => 0 11111 1000000000
+var UINT16_PINF = 0x7C00;                   // 0x7C00 = 31744 => 0 11111 0000000000
+var UINT16_NINF = 0xFC00;                   // 0xFC00 = 64512 => 1 11111 0000000000
+var UINT16_POSITIVE_ZERO = 0x0000;          // 0x0000 = 0 => 0 00000 0000000000
+var UINT16_NEGATIVE_ZERO = 0x8000;          // 0x8000 = 32768 => 1 00000 0000000000
+var FLOAT32_MANTISSA_MASK = 0x7FFFFF;       // 0x7FFFFF = 8388607 => 0 11111111 11111111111111100000000
+var FLOAT32_IMPLICIT_BIT = 0x800000;        // 0x800000 = 8388608 => 1 00000000 00000000000000000000000
+var FLOAT32_SUBNORMAL_STICKY_MASK = 0x1FFF; // 0x1FFF = 8191 => 0 00111111 11111110000000000000000
```

Comment:
I don't think you want to show this as a float32 binary string, but rather a uint32 string as this value is used for masking the significand.

```
In [20]: base.toBinaryStringUint32( 0x1FFF )
Out[20]: '00000000000000000001111111111111'
```